### DISCUSSION TITLE
Might it be a good idea for helm-lib.el to require ffap?

### DISCUSSION THREAD

**Discussion by PalaceChan** [emacs-helm/helm#2588] [Q&A] [answered]
after this bugfix: https://github.com/emacs-helm/helm/issues/2574, `helm-guess-filename-at-point` let binds a few ffap variables. This was triggering "Defining as dynamic an already lexical var" when trying to use helm after startup (which would go away after some other action defined it, in my case opening an org file). I "fixed" by calling `(require 'ffap)` in the `:config` part of `(use-package helm` and wondering if this simple change makes sense or if it should stay in user init.el

**Comment by thierryvolpiatto** [comment, marked as answer]
Palace ***@***.***> writes:

> after this bugfix: #2574, helm-guess-filename-at-point let binds a few
> ffap variables. This was triggering "Defining as dynamic an already
> lexical var" when trying to use helm after > startup (which would go
> away after some other action defined it, in my case opening an org
> file). I "fixed" by calling (require 'ffap) in the :config part of
> (use-package helm and > wondering if this simple change makes sense or
> if it should stay in > user init.el

Yes you are right, I forgot to declare ffap vars in helm-lib, it is now
done, let me know if it fix this bug for you.
Thanks.



>
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.*Message ID: ***@***.***>


-- 
Thierry
